### DISCUSSION TITLE
feat: Add language support for color code

### DISCUSSION THREAD

**Discussion by Jisan09** [athul/waka-readme#126] [Ideas]
@athul Can add an extra env variable (ie. CODE_LANG) to change the language for the code to get colored text.

Currently its working as `txt` to write the output in plain text , if add an env to change that we can use language like `python` or `ruby` to get colored output like this

![image](https://github.com/athul/waka-readme/assets/66010486/b0d8235e-0435-4b1b-b68e-f96474b73062)

**Comment by athul** [comment]
It'd be great of you could open a PR for this. I'm unsure if it's possible to color text in python. Maybe with CSS? I'm not exactly sure how to do it.

@joe733 what do you think?

**Reply by Jisan09** [reply]
okay i'll open a pr for this

**Reply by nandgator** [reply]
Looks good.
